examples/java-agent/build.gradle

```diff
@@ -0,0 +1,54 @@
+plugins {
+  id 'java'
+  id 'com.google.cloud.tools.jib' version '0.9.11'
+  id 'de.undercouch.download' version '3.4.0'
+}
+
+ext {
+  // where to download the Stackdriver Debugger agent https://cloud.google.com/debugger/docs/setup/java
+  stackdriverDebuggerAgentUrl = 'https://storage.googleapis.com/cloud-debugger/compute-java/debian-wheezy/cdbg_java_agent_gce.tar.gz'
```

Comment:
if we use gce agent, we don't need to generate the service account at all. just make sure the cluster has the proper scope.